server/routes.go

```diff
@@ -1451,6 +1452,38 @@ func (s *Server) PsHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, api.ProcessResponse{Models: models})
 }
 
+func (s *Server) TemplateHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
`handleTemplate`

There is no need to export this, and prefixing standardizing on `handleX` (like in ollama.com, et al, we can quickly distinguish http handlers from other functions, almost always ending at `handleX(`.

Reply:
yeah very fair - 
a) I was following the pattern for rest of the routes but was going to bring up breaking away from that and if that's okay to go about (seems like it is)
b) Makes sense if okay with having differences in route naming